docs/en_US/reference/emqxBroker.md

```diff
@@ -0,0 +1,445 @@
+
+    *ResourceRequirements describes the compute resource requirements.*
+
+    - **storage.resources.limits** (map[string][Quantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/#Quantity))
+
+      Limits describes the maximum amount of compute resources allowed. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+
+    - **storage.resources.requests** (map[string][Quantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/#Quantity))
+
+      Requests describes the minimum amount of compute resources required. If Requests is omitted for a container, it defaults to Limits if that is explicitly specified, otherwise to an implementation-defined value. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+
+  + **storage.volumeName** (string)
+
+    VolumeName is the binding reference to the PersistentVolume backing this claim.
+
+  + **storage.storageClassName** (string)
+
+    Name of the StorageClass required by the claim. More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#class-1
+
+  + **storage.volumeMode** (string)
+
+    volumeMode defines what type of volume is required by the claim. Value of Filesystem is implied when not included in claim spec.
+
++ **extraVolumes**([]Volume)
+
+  *Patch strategies: retainKeys, merge on key `name`*
+
+  List of volumes that can be mounted by containers belonging to the pod. More info: https://kubernetes.io/docs/concepts/storage/volumes
+
++ **extraVolumeMounts** ([]VolumeMount)
+
+  *Patch strategy: merge on key `mountPath`*
+
+  Pod volumes to mount into the container's filesystem. Cannot be updated.
+
+  *VolumeMount describes a mounting of a Volume within a container.*
+
+  - **extraVolumeMounts.mountPath** (string), required
+
+    Path within the container at which the volume should be mounted. Must not contain ':'.
+
+  - **extraVolumeMounts.name** (string), required
+
+    This must match the Name of a Volume.
+
+  - **extraVolumeMounts.mountPropagation** (string)
+
+    mountPropagation determines how mounts are propagated from the host to container and the other way around. When not set, MountPropagationNone is used. This field is beta in 1.10.
+
+  - **extraVolumeMounts.readOnly** (boolean)
+
+    Mounted read-only if true, read-write otherwise (false or unspecified). Defaults to false.
+
+  - **extraVolumeMounts.subPath** (string)
+
+    Path within the volume from which the container's volume should be mounted. Defaults to "" (volume's root).
+
+  - **extraVolumeMounts.subPathExpr** (string)
+
+    Expanded path within the volume from which the container's volume should be mounted. Behaves similarly to SubPath but environment variable references $(VAR_NAME) are expanded using the container's environment. Defaults to "" (volume's root). SubPathExpr and SubPath are mutually exclusive.
+
+## EmqxBrokerTemplate
+
```

Comment:
Maybe attach the link of emqx docs which related with the field  better?

Reply:
There is no relevant emqx documentation for `EmqxBrokerTemplate`